dockerfiles/agent/ubi.Dockerfile

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+#
+# Copyright (c) 2019-2020 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+
+FROM registry.redhat.io/ubi8/ubi-minimal:8.3
+
+ENV USER=user \
+    UID=12345 \
+    GROUP=group \
+    GID=23456
+
+ADD content_sets_centos8.repo /etc/yum.repos.d/
+
+#cron task not work in openshift in case https://github.com/gliderlabs/docker-alpine/issues/381
+#so will used  supercronic https://github.com/aptible/supercronic
+ENV SUPERCRONIC_URL=https://github.com/aptible/supercronic/releases/download/v0.1.9/supercronic-linux-amd64 \
+    SUPERCRONIC=supercronic-linux-amd64 \
+    SUPERCRONIC_SHA1SUM=5ddf8ea26b56d4a7ff6faecdd8966610d5cb9d85
+
+COPY cron/backup-cron-job /etc/crontabs/backup-cron-job
+COPY scripts /scripts
+#
+# Add user that will be able to start watcher binary but nothing more
+# the result will be propagated then into scratch image
+# See https://stackoverflow.com/a/55757473/12429735RUN
+#
+RUN microdnf update -y \ 
+      && microdnf install -y \
```

Comment:
please follow convention of && at end of lines or beginning of lines

Reply:
I do have && at beginning of lines, it's the same as `dockerfiles/agent/Dockerfile`?

Reply:
Ok maybe I read the only wrong line
```
&& mkdir  /var/run/sshd && \
```